src/partials/header.html

```diff
@@ -0,0 +1,39 @@
+<div class="header">
```

Comment:
Sensowniej wydaje mi się byłoby użyć tagu <header>, a niżej <div class="container header">. Lepsza semantyka, ale to czepianie się.